### ISSUE TITLE
Is there a property to know if I will attend the appointment (the "going" stuff)?

### ISSUE THREAD

**Issue by diegosanchez** [kidd/org-gcal.el#111]
I would like to know if the appointment will be attended from org-mode. Is there any way to know it?

I'm not "going" so far:

![Screen Shot 2020-11-02 at 12 04 40](https://user-images.githubusercontent.com/3934479/97883428-9f522400-1d03-11eb-9e1d-46aaa8e273b5.png)

I'm going (I accepted/confirmed the appointment):

![Screen Shot 2020-11-02 at 12 05 48](https://user-images.githubusercontent.com/3934479/97883564-c90b4b00-1d03-11eb-96bb-a4bf114c026f.png)

**Comment by telotortium** [comment]
No, that would be a feature request. Feel free to implement it if you want.

**Comment by m01** [comment]
I think the "not going", i.e. you've declined the meeting, would be the most interesting-to-know case. I guess declined events could be marked similarly to cancelled events with a(nother) todo keyword.
